### DISCUSSION TITLE
Fitbit finally closed down the Dashboard discussions

### DISCUSSION THREAD

**Discussion by TLeafTO** [jlai/fitness-dashboard#77] [General]
It's understandable that they closed the discussion about ending the Fitbit Web Dashboard. There were about 1500 or more posts in it. Most were members complaining and disparaging Google/Fitbit.
It was, however also a place where some of us could share the good news about the  Fitness Dashboard that Jay created.
Sadly, many Fitbit users will now not know about it.

**Comment by mscatz-9** [comment]
They do have a section for 3rd party integrations.  Do you think they would be opposed to a post about the exercise quest dashboard there?

[https://community.fitbit.com/t5/Third-Party-Integrations/bd-p/other-apps](url)

**Comment by TLeafTO** [comment]
Probably. We have started to share again in Fitbit model specific areas. I have an Inspire 3
https://community.fitbit.com/t5/Inspire-3/Dashboard/m-p/5675549#M11344
